src/SmartFormat/SmartFormatter.cs

```diff
@@ -149,9 +161,22 @@ public string Format(string format, params object[] args)
         /// <returns>Returns the formatted input with items replaced with their string representation.</returns>
         public string Format(IFormatProvider? provider, string format, params object[] args)
         {
-            var output = new StringOutput(format.Length + args.Length * 8);
+            return Format(null, args, format);
```

Comment:
```suggestion
            return Format(provider, args, format);
```
This resolves failing unit tests